### ISSUE TITLE
Lets add a Travis Mac Build

### ISSUE THREAD

**Issue by original-brownbear** [Proemion/Helenus#5]
We should have Travis run this build on Mac too. 
Likely contributors will be using Mac down the line and while Travis provides us with relatively speedy builds, it is not a replacement for running unit-tests locally with second long feedback loops.
=> We should keep the build Mac portable. Only issue is, we need Oracle JDK8 for the Cassandra. Travis does not supply this at present for Mac, so we will have to script this ourselves.

# Issue is resolved once:

* We have a working Mac build on Travis alongside the existing Linux one

**Comment by original-brownbear** [comment]
@drempapis since you mentioned having this work fine on a Mac, wanna deal with this one real quick ? :)

**Comment by drempapis** [comment]
@original-brownbear 

Yes sure. I first checked Travis last Friday, as it is a dependency of the project. Give me please some general overview of what has to be done. I can start on Tuesday afternoon, as it is bank holiday on Monday here and I ll be out.

**Comment by original-brownbear** [comment]
@drempapis for this issue all you need to do is adjust the `.travis.yml` to make it work with Mac too.
The only thing preventing this from working out of the box is, that Travis has no Oracle JDK8 on Mac available it seems, so that installation will have to be scripted I'm afraid.
But other than that, this should be a quick and easy task to get you started and give some insight into the dependencies of this project.
=> Just ping me here though, if I missed the point of your question entirely :D :)

**Comment by drempapis** [comment]
@original-brownbear 
I forked the project and created a branch in the local repository, but next I couldn't create back the branch in the original one. 

As for this issue it needs to be executed on Travis, there should be a branch on the original repo.

I tried the conbination of osx - xcode8 in another project running on travis and it worked.

I didn' t mean to make a pull request at this moment. I was trying to make a build on Travis.

**Comment by drempapis** [comment]
@original-brownbear 
A quick approach that worked for me, having a Dockerfile with oracle-java8 and maven3.3.9, but without caching. Any opinion?

services:
  - docker
script: 
 - docker build -t helenus .
 - docker run -it -v $(pwd):/root/sources -w /root/sources helenus mvn clean install

**Comment by drempapis** [comment]
@original-brownbear 
Yes it does, I just tried the last approach using docker in another personal project, as I cannot make a branch directly to Helenus and I didn't want to do a pull request each time the script was changed.

How do you propose to continue on that? :)
